vespajlib/src/main/java/com/yahoo/slime/ArrayValue.java

```diff
@@ -36,15 +40,22 @@ public void prepareFor(ArrayValue self, Type type) {
                 self.impl = new GenericImpl(this, self.used);
             }
         }
+        private static long [] grow(long [] arr) {
```

Comment:
I think the type should not contain space: long[] (similar for other grow functions)